lang/rs/wireguard/patch/0007-Use-CNDP-CNE-Rust-library.patch

```diff
@@ -2050,7 +2050,7 @@ index 00a9d42..0000000
 -    include_directories : cndp_inc)
 -
 -jcfg_parse_libname = 'rust_' + 'jcfg_parse'
--librust_jcfg_parse = both_libraries(jcfg_parse_libname, jcfg_parse_sources, install: false, dependencies: [cndp_dep])
+-librust_jcfg_parse = library(jcfg_parse_libname, jcfg_parse_sources, install: false, dependencies: [cndp_dep])
```

Comment:
We shouldn't ideally edit multiple lines in a patch file since patch might not apply. Better to create a new patch instead of editing existing patch file.

Reply:
@manojgop I believe if you add/remove lines in a patch file then patch does not like it as the >@@ lines are wrong now. If you could please verify these patches will apply it would be great thanks. If they do not then I will figure out the next step.

Reply:
> @manojgop I believe if you add/remove lines in a patch file then patch does not like it as the >@@ lines are wrong now. If you could please verify these patches will apply it would be great thanks. If they do not then I will figure out the next step.

The wireguard patches applied without error and CNE rust apps also builds fine. But I tested only with shared library.

Reply:
Thank you, shared or static libs should not make a difference.